Client/Frontend/Login Management/LoginListViewModel.swift

```diff
@@ -63,7 +63,7 @@ final class LoginListViewModel {
         profile.logins.searchLoginsWithQuery(query) >>== { logins in
             var log = logins.asArray()
             log.append(LoginRecord(fromJSONDict: ["hostname" : "twitter.com", "timePasswordChanged": 46800000]))
-            log.append(LoginRecord(fromJSONDict: ["hostname" : "abreach.com", "timePasswordChanged": 46800000, "username": "username"]))
+            log.append(LoginRecord(fromJSONDict: ["hostname" : "duisatconsecteturloremdonecmassasapienfaucibusetmolestieacfeugiatsedlectusvestibulummattisullamcorpervelitsedullamcorp.com", "timePasswordChanged": 46800000, "username": "username"]))
```

Comment:
So that we can submit this feature build for design and QA review, could we add a few domains some one is likely not to have in their logins (like blockbuster or something), that are real websites?  We don't need to keep fake ones for now since we have that issue open on the prod version of that app about non real urls not showing errors correctly.